data/ecosystems/c/celo.toml

```diff
@@ -118,12 +122,16 @@ sub_ecosystems = [
   "Tatum",
   "Toucan Protocol",
   "Tradegen",
+  "TrueFeedBack",
   "Ubeswap",
   "Unifty",
   "Uniswap",
   "untangled.finance",
   "Valora",
+  "Wagmi",
+  "WalletConnect",
```

Comment:
we keep WalletConnect as general with no sub-ecos since it covers all crypto, please remove

Reply:
Does it support non-EVM chains?

Reply:
Removed